File: python/7 kyu/177.py
# https://www.codewars.com/kata/56459c0df289d97bd7000083/train/python

# Passed

def generator(_from, _to, _step):
    if _step == 0:
        return []
    
    if _from > _to:
        _step = -_step
        _to -= 2
        
    result = list(range(_from, _to + 1, _step))
    return result

output = generator(20, 10, 1)
print(output)

File: nodejs/7 kyu/177.js
// https://www.codewars.com/kata/57eeb8cc5f79f6465a0015c1/train/javascript

// Passed

function isKiss(words) {
  let allWords = words.split(" ");
  let totalWords = allWords.length;
  let isSimple = allWords.every((word) => word.length <= totalWords);
  let result = isSimple ? "Good work Joe!" : "Keep It Simple Stupid";
  return result;
}

const output = isKiss("Joe had a bad day");
console.log(output);